.github/workflows/commit-to-main.yml

```diff
@@ -0,0 +1,87 @@
+name: Trigger basic downstream CI
+
+permissions:
+  contents: read
+
+on:
+  push:
+    branches: [ "main" ]
+
+jobs:
+  trigger-downstream-ci:
+    strategy:
+      fail-fast: false
+    runs-on: ubuntu-latest
+    steps:
+      # TODO: missing projects?
+      - name: Trigger OQS-OpenSSL CI
+        run: |
+          curl --silent \
+               --write-out "\n%{response_code}\n" \
+               --user ${BUILD_TRIGGER_TOKEN}: \
+               --request POST \
+               --header "Content-Type: application/json" \
+               --data '{ "branch": "OQS-OpenSSL_1_1_1-stable", "parameters": { "run_downstream_tests": true } }' \
```

Comment:
Doesn't look necessary to keep this one

Reply:
Same comment as below re liboqs-rust applies here.